internal/build/templates/k8s-config-installer.service.tpl

```diff
@@ -8,6 +8,8 @@ Type=oneshot
 TimeoutSec=900
 Restart=on-failure
 RestartSec=60
+# TODO (atanasdinov): Fix context of files and directories instead of only forcing a rebuild and reload of policies
+ExecStartPre=/bin/sh -c "semodule -B"
```

Comment:
would `semodule --refresh` be sufficient? e.g. why do we need an unconditional rebuild and reload?